nixos/modules/services/web-apps/calibre-web.nix

```diff
@@ -121,6 +121,7 @@ in
         ]
         ++ optional (cfg.options.calibreLibrary != null) "config_calibre_dir = '${cfg.options.calibreLibrary}'"
         ++ optional cfg.options.enableBookConversion "config_converterpath = '${pkgs.calibre}/bin/ebook-convert'"
+        ++ optional cfg.options.enableBookConversion "config_binariesdir = '${pkgs.calibre}/bin/'"
```

Comment:
We can combine this with the above list by using lib.optionals

Reply:
👍  Change applied, thanks for the recommendation.